### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by ish12321** [theAsmodai/metamod-r#78] [Q&A] [answered]
Shall i use this or latest metamod p with rehlds , regamedll_cs

**Comment by theAsmodai** [comment]
Please wait for release.

**Comment by ish12321** [comment]
OK

**Comment by ish12321** [comment]
WHEN WILL IT BE READY FOR RELEASE ?

**Comment by hajimura** [comment]
Asap

**Comment by ish12321** [comment]
Ready?

**Comment by ish12321** [comment]
BUMP ! READY ?

**Comment by stamepicmorg** [comment, marked as answer]
@ish12321 ready. u can try to use it with rehlds and regame. and do feedback here. if u want. why not.
